### PR TITLE
feat(slack): support thread-timestamp

### PR DESCRIPTION
**Pull request by v1v**
I copied partially https://github.com/elastic/apm-pipeline-library/tree/main/.github/actions/slack-message so we can support the `thread-timestamp` output.

Notifies https://github.com/elastic/apm-server/pull/13307

image

### CODE REVIEW

File: slack/send/README.md [hunk 1]
```diff
@@ -2,11 +2,23 @@
...
+| `bot-token`  | Specify the slack bot token.                                                                                                                                                                                                                                                                                                                   | `true`   | ` `     |
+| `channel-id` |                                                                                                                                                                                                                                                                                                                                                | `true`   | ` `     |
+| `message`    | Slack message on Markdown format. Multiline messages must be escaped using URL encoding.
+https://github.com/orgs/community/discussions/26288
```
Comment (v1v) [inline slack/send/README.md:10]: as far as i see the automation does not support a multline approach, though GitHub input descriptions allow so.

**Review by reakaleek — APPROVED** [review]

File: slack/send/README.md (hunk 1 above)
Comment (v1v) [reply slack/send/README.md:10]: @reakaleek , what are your thoughts regarding multiline descriptions and auto-docs generation?

Comment (reakaleek) [reply slack/send/README.md:10]: Oh. Good point. Let me take a quick look at this.

Comment (v1v) [reply slack/send/README.md:10]: Thanks Jan, btw, there is no urgency, those changes can be supported in a follow-up as they are cosmetic IMO

Comment (reakaleek) [reply slack/send/README.md:10]: I found a simple fix for this.. but needs a new release in reakaleek/gh-action-readme. I only need to replace `\n` with `<br>` and the table will be look as expected

Comment (reakaleek) [reply slack/send/README.md:10]: https://github.com/elastic/oblt-actions/pull/32 should fix it

**Pull request merged**